modules/base/rotation/staticrotation.cpp

```diff
@@ -59,7 +59,14 @@ namespace {
 
     struct [[codegen::Dictionary(StaticRotation)]] Parameters {
         // Stores the static rotation as a vector containing Euler angles, a quaternion
-        // or a rotation matrix
+        // representation, or a rotation matrix.
+        // For the Euler angles, the values have to be provided in radians. To convert
```

Comment:
How will these linebreaks look in the resulting part on the docs page?

Reply:
I added one more extra linebreak to prepare it for the future.  For now, all of the newlines don't do anything and the text gets compressed.  But we'll want to have those newlines be respected and turned into paragraphs.

Right now:
![image](https://github.com/OpenSpace/OpenSpace/assets/4921724/1d77a4d9-0c21-43e2-a6d2-2991a8584898)